### PR TITLE
enhancement: merge welcome and legal

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/inputs/Checkbox.svelte [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-    import { Icon, Text } from 'shared/components'
+    import { Icon, Text, TextType } from 'shared/components'
```
Comment (maxwellmattryan) [inline packages/shared/components/inputs/Checkbox.svelte:2]: `@ui`

File: packages/shared/lib/core/app/constants/index.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 export * from './default-app-settings.constant'
 export * from './external-allowed-links.constant'
+export * from './privacy-policy-link.constant'
```
Comment (maxwellmattryan) [inline packages/shared/lib/core/app/constants/index.ts:3]: We should use `URL` in the naming as we've never used "link"

File: packages/shared/lib/core/app/constants/privacy-policy-link.constant.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const PRIVACY_POLICY_LINK = 'https://firefly.iota.org/privacy'
```
Comment (maxwellmattryan) [inline packages/shared/lib/core/app/constants/privacy-policy-link.constant.ts:1]: See naming comment above ^

File: packages/shared/lib/core/app/constants/terms-of-service-link.constant.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export const TERMS_OF_SERVICE_LINK = 'https://firefly.iota.org/terms'
```
Comment (maxwellmattryan) [inline packages/shared/lib/core/app/constants/terms-of-service-link.constant.ts:1]: See naming comment above ^

**Review by maxwellmattryan — COMMENTED** [review]
Have a few requested changes :pray:

File: packages/desktop/views/onboarding/views/app-setup/AppSetupRouter.svelte [hunk 5]
```diff
@@ -1,17 +1,13 @@
 <script lang="ts">
-    import { Transition } from 'shared/components'
-    import { LanguageAndAppearanceView, LegalView, WelcomeView } from './views'
     import { AppSetupRoute, appSetupRoute } from '@core/router'
+    import { Transition } from 'shared/components'
```
[the remark this answers is not in the document]
Comment (nicole-obrien) [reply packages/desktop/views/onboarding/views/app-setup/AppSetupRouter.svelte:3]: ahh i just organised existing imports 🙄 😆

File: packages/desktop/views/onboarding/views/app-setup/views/WelcomeView.svelte [hunk 6]
```diff
@@ -1,38 +1,61 @@
...
+        </Text>
     </div>
-    <div slot="leftpane__action">
-        <Button onClick={onContinueClick} classes="w-full">{localize('actions.continue')}</Button>
+    <div slot="leftpane__action" class="flex flex-col space-y-8">
+        <Checkbox bind:checked={termsAccepted}>
+            <Text slot="label" type={TextType.p} secondary>
+                I've read and I accept the <Link onClick={onTermsOfServiceClick}>Terms of Service</Link> and <Link
+                    onClick={onPrivacyPolicyClick}>Privacy Policy</Link
+                >
```
Comment (Tuditi) [inline packages/desktop/views/onboarding/views/app-setup/views/WelcomeView.svelte:51]: Shouldn't this be localized?

**Review by Tuditi — CHANGES_REQUESTED** [review]
One change requested, otherwise works fine! :relaxed:

File: packages/desktop/views/onboarding/views/app-setup/views/WelcomeView.svelte (hunk 6 above)
Comment (Tuditi) [reply packages/desktop/views/onboarding/views/app-setup/views/WelcomeView.svelte:51]: As discussed on Discord, we don't know the locale at the start-up, so we have to rethink the localization of the welcome page if we want to support more languages.

**Review by Tuditi — APPROVED** [review]
:fire:

**Pull request merged**